package.json

```diff
@@ -14,13 +14,12 @@
     "lib/**/*.js"
   ],
   "dependencies": {
-    "@digitalbazaar/ecdsa-multikey": "^1.0.0",
+    "@digitalbazaar/ecdsa-multikey": "github:digitalbazaar/ecdsa-multikey#fix-sign-and-verify-issues",
```

Comment:
TODO: Update this line after this branch has been deployed to the npm registry.

Reply:
This can be updated now.